maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/SurefireHelper.java

```diff
@@ -286,7 +289,15 @@ private static String createErrorMessage( SurefireReportParameters reportParamet
         }
         else
         {
-            msg.append( "There are test failures.\n\nPlease refer to " )
+            if ( result.getFailures() > 0 )
+            {
+                msg.append( "There are test failures." );
+            }
+            else
+            {
+                msg.append( "There are too many flakes." );
```

Comment:
@oehme 
I think this would not be printeted very often even if it could because there is if-else.
We do not have IT test but both messages could be printed along. Why the message `There are too many flakes.` does not have the concrete numbers. We have both numbers available and they are the cause why the message is going to be printed. I only want to give the user all reason why the plugin crashed.

Reply:
Done, both messages are reported now if both conditions are true. Also, the number of flakes and failOnFlakeCount are reported now.

Reply:
@oehme 
Thx, a good job!